packages/pancake-profile-sdk/src/constants/contracts.ts

```diff
@@ -0,0 +1,54 @@
+export default {
+  masterChef: {
```

Comment:
Perhaps we should move the contract address and ABIs to its own package. @RabbitDoge, @ChefHutch ?